### PR TITLE
upload all animation example document

### PR DESCRIPTION
**Pull request by yulijia**
1. upload all animation example document
2. figures are in Dropbox
3. I will modify mistakes in each article in next week.

### CODE REVIEW

**Comment by yulijia** [comment]
1. change animation interval
2. remove irrelevant interactive animation 
3. modify documents.

done.

File: content/animation/example/BM-circle.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ 
+
+```r
+library(animation)
+ani.options(interval = 0.1, nmax = 100)
+par(mar = rep(0.5, 4))
+BM.circle(cex = 2, pch = 19)
+```
+
+<video controls loop autoplay><source src="/figures/animation/example/BM-circle/demo-a.mp4" /><p>plot of chunk demo-a</p></video>
```
Comment (yihui) [inline content/animation/example/BM-circle.md]: There needs to be `https://assets.yihui.name` before `/figures/animation/...`, but I can add it by myself later. Thanks!

File: content/animation/example/BM-circle.Rmd [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: BM.circle()
+subtitle: Brownian Motion in a Circle
+date: '2017-04-04'
+slug: BM.circle
```
Comment (yihui) [inline content/animation/example/BM-circle.Rmd]: Could you replace `.` with `-` in slugs?

**Review by yihui — CHANGES_REQUESTED** [review]
This is excellent! Thanks a lot!

File: content/animation/example/BM-circle.Rmd (hunk 2 above)
Comment (yulijia) [reply content/animation/example/BM-circle.Rmd]: sure. no problem

**Review by yihui — APPROVED** [review]
Perfect!

**Pull request merged**